tests/scripts/run-xunit-perf.cmd

```diff
@@ -107,8 +107,12 @@ setlocal
   )
 
   rem Save off the results to the root directory for recovery later in Jenkins
-  call :run_cmd xcopy "Perf-%BENCHNAME%*.xml" "%CORECLR_REPO%\" || exit /b 1
-  call :run_cmd xcopy "Perf-%BENCHNAME%*.etl" "%CORECLR_REPO%\" || exit /b 1
+  IF EXIST "Perf-%BENCHNAME%.xml" (
```

Comment:
I would suggest keeping `xcopy` with the following flags:
```cmd
xcopy.exe /VYQ
/V           Verifies the size of each new file.
/Y           Suppresses prompting to confirm you want to overwrite an
               existing destination file.
/Q           Does not display file names while copying.
```
and always append either the `COLLECTION_FLAGS` or using the `ETW_COLLECTION` values, though I am more incline to use the `ETW_COLLECTION` flag: `Perf-%BENCHNAME%-Profile_%ETW_COLLECTION%.xml`. The reason is that if the collection flags ever add a character that is invalid for a file name, this would break.

Another fun fact is that the `COPY` command uses the `+` character to append files (and the API uses that character to append providers):
```cmd
To append files, specify a single file for destination, but multiple files
for source (using wildcards or file1+file2+file3 format).
```